packages/core/strapi/src/cli/commands/cloud/config/local.ts

```diff
@@ -0,0 +1,15 @@
+import path from 'path';
+import os from 'os';
+import fs from 'fs';
+
+export const TMP_DIR = path.join(os.tmpdir(), 'strapi-cli');
```

Comment:
Is this safe to store here? Wouldn't it mean that every user on the machine is logged in if one user logged in?

Can we use `os.homedir()` and set permissions to only be read by the owner?

Reply:
I like this idea. I agree that even if we cannot prevent token exposure, restricting usage to the owner can prevent accidental misusage of the CLI.

Reply:
I guess using homedir won't solve this problem. So files will never be deleted from the tmp folder.
We need to take care of this permission thing, maybe create a subfolder per user and apply appropriate permissions to only have the current user accessing it in read mode 🤔

Reply:
It looks like they use fs permissions 

```
    return writeJSON.sync(AUTH_CONFIG_FILE_PATH, authConfig, {
      indent: 2,
      mode: 0o600,
    });
 ```

Reply:
a lot of people use configstore for these kind of things also not sure what method they use to pick a folder I believe it's just homedir